src/pages/LendingTeams/TeamLeaderboard.vue

```diff
@@ -0,0 +1,194 @@
+										>
+											{{ leaderboardTeam.team.name }}
+										</div>
+									</router-link>
+								</div>
+								<div class="tw-flex-none tw-ml-1">
+									<template v-if="!isNewMembers">
+										{{ numeral(leaderboardTeam.value).format('$0,0a') }}
+									</template>
+									<template v-else>
+										{{ numeral(leaderboardTeam.value).format('0,0a') }}
+									</template>
+								</div>
+							</div>
+							<div
+								class="tw-flex-none tw-w-full"
+							>
+								<kv-progress-bar
+									:aria-label="progressLabel"
+									:min="0"
+									:max="maxValue(timeFrame)"
+									:value="parseInt(leaderboardTeam.value)"
+								/>
+							</div>
+						</div>
+					</div>
+					<button
+						v-if="numberOfShownTeams == 4"
+						@click="expandTeams = true"
+						class="tw-text-link tw-m-auto tw-block tw-text-small"
+					>
+						Show more teams
+					</button>
+				</kv-tab-panel>
+			</template>
+		</kv-tabs>
+	</div>
+</template>
+
+<script>
+import numeral from 'numeral';
+import teamNoImage from '@/assets/images/team_s135.png';
+import _throttle from 'lodash/throttle';
+import KvTab from '~/@kiva/kv-components/vue/KvTab';
+import KvTabPanel from '~/@kiva/kv-components/vue/KvTabPanel';
+import KvTabs from '~/@kiva/kv-components/vue/KvTabs';
+import KvProgressBar from '~/@kiva/kv-components/vue/KvProgressBar';
+
+export default {
+	name: 'TeamLeaderboard',
+	components: {
+		KvProgressBar,
+		KvTabs,
+		KvTab,
+		KvTabPanel,
+	},
+	data() {
+		return {
+			numeral,
+			teamNoImage,
+			isMobile: false,
+			expandTeams: false,
+			selectedTabIndex: 0
+		};
+	},
+	props: {
+		boardType: {
+			type: String,
+			validator: value => ['memberCount', 'overallLoanedAmount'].includes(value),
+			required: true,
+		},
+		boardTeams: {
+			type: Object,
+			default: () => ({}),
+			required: true,
+		}
+	},
+	computed: {
+		isNewMembers() {
+			return this.boardType === 'memberCount';
+		},
+		numberOfShownTeams() {
+			if (this.isMobile && !this.expandTeams) {
+				return 4;
+			} return 10;
+		},
+		progressLabel() {
+			if (this.isNewMembers) {
+				return 'Total Amount of Members the Team has';
+			} return 'Total Amount the Team has lent';
+		}
+	},
+	methods: {
+		determineIfMobile() {
+			this.isMobile = document.documentElement.clientWidth < 735;
+		},
+		teamsToShow(timeFrame) {
+			const teams = this.boardTeams[timeFrame] ?? [];
+			return teams.slice(0, this.numberOfShownTeams);
+		},
+		maxValue(timeFrame) {
+			const teams = this.boardTeams[timeFrame] ?? [];
+			const values = teams.map(team => parseInt(team.value, 10));
+			return Math.max(...values);
+		},
+		handleTabChanged(index) {
+			this.selectedTabIndex = index;
+		},
+	},
+	mounted() {
+		this.determineIfMobile();
+		window.addEventListener('resize', _throttle(() => {
+			this.determineIfMobile();
+		}, 200));
+	},
+	beforeDestroy() {
+		window.removeEventListener('resize', _throttle(() => {
```

Comment:
The `throttle` needs to be set to a variable, for example in `RandomLoanSelector.vue`.